app/assets/javascripts/search.js

```diff
@@ -43,39 +82,31 @@ $(document).ready(function(){
     scrollUp()
   })
 
-  function scrollUp(){
-    $(window).trigger('scroll-to', [$('#search-form'), 10]);
-  }
-
-  function progressHtml() {
-    str = '<div class="progress"> \
-        <div class="progress-bar progress-bar-striped bg-danger" role="progressbar" \
-    style="width: 100%" aria-valuenow="100" aria-valuemin="0" aria-valuemax="100"></div> \
-      </div>'
-    return str
-  }
 
   function loadTranscripts(){
-    searchText = $('#keyword').val()
-    $(".transcript-list-search").html('<div class="loading"></div>')
     data = {
+      institution_id: institutionId,
       collection_id: collectionId,
+      sort_id: sortId,
       q: searchText,
-      deep: 1
+      theme: theme
```

Comment:
The strong params in the controller don't allow `sort_id` or `theme`, are they used/needed?